packages/phone-number-privacy/combiner/src/signing/get-threshold-signature.ts

```diff
@@ -69,16 +73,36 @@ async function requestSignatures(request: Request, response: Response) {
   const errorCodes: Map<number, number> = new Map()
   const blsCryptoClient = new BLSCryptographyClient()
 
+  const logger: Logger = response.locals.logger
+
   const signers = JSON.parse(config.odisServices.signers) as SignerService[]
   const signerReqs = signers.map((service) => {
     const controller = new AbortController()
     const timeout = setTimeout(() => {
       controller.abort()
     }, config.odisServices.timeoutMilliSeconds)
 
-    return requestSignature(service, request, controller)
+    const obs = new PerformanceObserver((list, observer) => {
+      logger.info(
+        { latency: list.getEntries()[0].duration, signer: service },
```

Comment:
Is this guaranteed to have at least one entry?

Reply:
Yes because the callback only gets called when an entry is added